src/vs/languages/sass/common/parser/sassParser.ts

```diff
@@ -53,6 +53,11 @@ export class SassParser extends cssParser.Parser {
 
 	// Sass variables: $font-size: 12px;
 	public _parseVariableDeclaration(panic:scanner.TokenType[]=[]): nodes.VariableDeclaration {
+		var cssVariableDeclaration= super._parseCssVariableDeclaration(panic);
```

Comment:
That looks wrong. Not all places where you can declare a sass variable are also suited to declare a css variable.